server/odc-test/src/main/java/com/oceanbase/odc/test/database/TestDBConfigurations.java

```diff
@@ -80,6 +103,10 @@ public TestDBConfiguration getTestMysqlConfiguration() {
         return connectType2ConfigurationMap.get(TestDBType.MYSQL);
     }
 
+    public TestDBConfiguration getTestDorisConfiguration() {
```

Comment:
Thanks for contributing code! We will soon be supporting the Doris database in unit tests for GitHub Action.